core/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/EwmaModel.java

```diff
@@ -52,6 +52,22 @@ public EwmaModel(double alpha) {
     }
 
     @Override
+    public boolean canBeMinimized() {
+        return true;
+    }
+
+    @Override
+    public MovAvgModel neighboringModel() {
+        double alpha = Math.random();
```

Comment:
Welllllll, "neighboring" in simulated annealing just means one of the params changes.  And if you only have one param... :)

But yes, I agree.  There is a variation that does exactly that: reduce the amount it jumps from current value based on temperature.  I tried that originally but had a hard time tuning it (reduce too fast/slow and you miss optimal).

Definitely something that I want to improve in the future though.  There are other methods which have rates per-coefficient, because each coefficient has a drastically different effect on the output, so they shouldn't be optimized at the same rate.